sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/MethodsShouldNotHaveIdenticalImplementations.cs

```diff
@@ -40,6 +40,20 @@ void Foo5()
         }
 
 
+        int DiffBySignature1(int arg1)
```

Comment:
New test case added to C# to clarify the existing behaviour - currently, the rule will report if the method bodies are the same, even if the method signatures are different.